### ISSUE TITLE
[BUG]: on scroll through home page invalid message

### ISSUE THREAD

**Issue by sambitraze** [Kide-Devs/Kide#41]
## **Describe the bug**
On trying to scroll the app should show progress bar instead of "that's all for now folks", the message should be shown only when we reach the end of list.

## **To Reproduce**
1. Go to Homescreen 
2. Click on on any tab (News/Blog/Events)
3. Scroll down at any speed
4. See error("that's all for now folks")

## **Expected behavior**
 A loader should apear while pulling data .

**Comment by StrangeNoob** [comment]
Hi @sambitraze and @Cryptonex7, if anyone isn't working on this issue, I would like to work on it.

**Comment by Cryptonex7** [comment]
Sure @StrangeNoob, We would love to have you contributing to Kide. 

Feel free to take this issue up, we will be awaiting your PR.

Regard, 
Cryptonex7

**Comment by Cryptonex7** [comment]
Hello @StrangeNoob,

Any updates on this would be appreciated.

Thanks.

**Comment by kamal-kaur04** [comment]
Hii @Cryptonex7, Can I work on this issue if it's not solved yet?

**Comment by Cryptonex7** [comment]
@kamal-kaur04 Yes Kamal, you can start with this, as we haven't recieved solutions for this yet.

Thanks and we will be awaiting your PR.

**Comment by shathurshan** [comment]
Hiii I am interested can you assign me

**Comment by Cryptonex7** [comment]
@shathurshan sure, feel free to work on this. The best PR will be merged, but all the PRs will be counted for Hacktoberfest.
